content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -247,20 +262,85 @@ zones, or regions.
     caption="A service mesh after a control plane instance fails"
     >}}
 
-The following list ranks control plane deployment examples by availability:
+### Independent Control Planes
+
+You can also deploy a control plane per cluster, with a shared root CA. Each cluster's control plane
+manages only the workloads in its own cluster. If the control plane is unavailable (due to upgrade or
+temporary outage), the workloads will continue to function with the existing sidecar configuration and
+will re-attempt to connect to the local control plane periodically.
+
+This model affords the following benefits:
+
+- Highest level of availability: If a control plane becomes unavailable, the scope of
+  the outage is limited to only that cluster and the workloads within it.
+
+- Maximum configuration isolation: You can make configuration changes in one cluster without impacting
+  workloads deployed in any other cluster.
+
+The following list ranks control plane deployment examples by availability, configuration isolation:
+
+- One control plane per region (**lowest availability**, **lowest isolation**)
+- Multiple control planes per region
+- One control plane per zone
+- Multiple control planes per zone
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. A single mesh can span
+[one or more clusters](#cluster-models) and [one or more networks](#network-models).
+All workloads in a mesh share a common certificate authority or the same trust
+domain depending on the following situations:
+
+- If you are using a vendor hosted CA that is shared across different customers,
+  then it is imperative to use a single trust domain for the entire mesh, as the
+  root CA across different customers could be the same (i.e. the vendor's CA itself).
+
+- If you have your own root CA for all your clusters, you can choose to use a single shared
+  trust domain or a separate trust domain for each cluster.
```

Comment:
Do we support separate trust domain for a single mesh?  Any examples?  Looking at our replicated control plane doc, we are using same root CA, and configuring auto_passthrough on the cluster gateways, so it is still single trust domain imho.

Reply:
Or you consider our replicated control plane is different trust domains with same root CA?

Reply:
we certainly support separate trust domains per cluster but with same CA. Heck, we are running this in production at several customer sites. its there in our docs - some obscure trust domain flag/field that people can set at control plane level.

Reply:
Thanks found it in meshconfig trust domain.